project.clj

```diff
@@ -13,7 +13,8 @@
   :dependencies [[nrepl/bencode "1.0.0"]]
 
   :aliases {"bump-version" ["change" "version" "leiningen.release/bump-version"]
-            "test-all" ["with-profile" "+1.7:+1.8:+1.9:+fastlane" "test"]}
+            "test-all" ["with-profile" "+1.7:+1.8:+1.9:+fastlane" "test"]
+            "docs" ["with-profile" "+maint" "run" "-m" "nrepl.impl.docs" "--lein"]}
```

Comment:
I saw this `--lein` here and in the code, but it doesn't seem to be used anywhere, so I assume it's some oversight.

Reply:
It's intentional, but there may be a better way to achieve what I want. I want `lein docs` to write to nREPL/doc/modules/ROOT/pages/op.adoc but I want lein with-profile +maint run nrepl.impl.docs to print to stdout by default.

The `--lein` flag is in the 'lein docs' alias definition then -main uses that as a short-circuit flag to write the output to the existing ops.adoc path instead of considering the `--file` param. This isn't very straightforward :/

Reply:
tbh didn't actually actually check if -main is invoke with cwd = the project root or if it's actually the cwd where lein is invoked from--assumed the latter. Will check....

Reply:
I see. I think think that `--lein` is a pretty confusing flag name, so it should probably be something else, but I don't have an amazing idea for this either.

Reply:
It turns out cwd is always the project root when calling a function via `lein run`, so in my latest revision I did away with `--lein` and just use `--file` with the relative path from the project root in the lein alias definition.